examples/computer_vision/efficientdet_pytorch/efficientdet_files/evaluator.py

```diff
@@ -0,0 +1,141 @@
+'''
```

Comment:
For any files in `efficientdet_files` directory you modified from the OG implementation, can you just make a quick comment at the top as to what the modifications were and why they were necessary?